avocado/utils/cpu.py

```diff
@@ -212,7 +212,7 @@ def get_family():
             family = zfamily_map[get_version()].lower()
         except KeyError as err:
             logging.warning("Could not find family for %s\nError: %s", get_version(), err)
-    elif arch == 'arm':
+    elif arch == 'arm' or arch == 'aarch64':
```

Comment:
Actually, this could be even more generic. I was debugging one problem today with Marcelo Condatta, and this was returning None. I just sent a patch here. We don't need this `or`.

#4239

Reply:
Thank you @beraldoleal, I will close this one.